src/eal.cpp

```diff
@@ -68,13 +67,54 @@ namespace eal {
   }
 
   // Postion Function Functions
-  vector<double> getPosition(vector<double> coeffs, double time) {
+  // vector<double> coeffs = [[cx_n, cx_n-1, cx_n-2, ... cx_0],
```

Comment:
Any opinions about the order I put these in? It's the opposite of the order that gsl uses. My reason for picking this order was just that I'm used to seeing equations written in this order: `2x^2 + 3x + 1 = 0`.

Reply:
I have to say, this is what I am used to seeing/using as well, but reading through GSL's documentation yesterday I saw they were opposite as well and found this interesting. I wonder if there is some standard...?

Reply:
One reason to use `1 + 3x + 2x^2` is because as you're iterating over the coefficients, you can use the index as your exponent.

```
for (int i = 0; i < coeff.size(), i++) {
  result += coeff[i] * pow(time, i);
}
```

Reply:
Another reason to put the constant first is you can arbitrarily pad with 0s without changing anything.

Reply:
@jessemapel Good point. Especially with the switch to allow any size, I'll go ahead and swap the order.